app/bower_components/angular-highlightjs/angular-highlightjs.js

```
@@ -0,0 +1,365 @@
+                    code = code[1];
+                  }
+                  else if (angular.isObject(code)) {
+                    // 1.0.7
+                    code = code.data;
+                  }
+
+                  code = code.replace(/^(\r\n|\r|\n)/m, '');
+                  ctrl.highlight(code);
+
+                  // Check if the highlight result needs to be compiled
+                  if (compileCheck && compileCheck(scope)) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)